src/components/atoms/DivideVector.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+const DivideVector = () => {
+  return <Divider></Divider>;
```

Comment:
이럴거면
```suggestion
  return <Divider />;
```

eslint 가 self-closing 관련해서 찡찡댔을텐데..?

Reply:
eslint 설정에 안넣어서 그런거같습니다... 수정하겠습니다